.github/workflows/experimental.yml

```diff
@@ -22,4 +26,4 @@ jobs:
       - run: ./scripts/pre-publish.sh --yes
         env:
           VERSION: '0.0.0-experimental.${{ github.sha }}'
-          DIST_TAG: experimental
+          DIST_TAG: ${{ github.event.inputs.dist-tag }}
```

Comment:
Since the versions only use the commit hash, they won't be ordered in any way. So if the intent is to do something like strapi@react18 you would have to delete the tag every time you want to release a newer version, correct?

Reply:
No i don't think that's true. You should still get whatever the "current" version the tag has been placed on based on publish timestamp iirc

Reply:
Ok, great. I do think the next step after this PR will be to start adding version numbers to any tag where it makes sense; which would be all of them except 'experimental' itself.